src/index.js

```diff
@@ -0,0 +1,127 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/js/fontawesome.js';
+import '@fortawesome/fontawesome-free/js/solid.js';
+import '@fortawesome/fontawesome-free/js/regular.js';
+import '@fortawesome/fontawesome-free/js/brands.js';
+import statusUpdate from './updateTodos.js';
+import updateTask from './updateTask.js';
+
+function component() {
+  let todos = [];
+  const todoInput = document.querySelector('.todo-input');
+  const todoList = document.querySelector('.todoList');
+
+  function saveLocalTodos(todo) {
+    if (localStorage.getItem('todos') === null) {
+      todos = [];
+    } else {
+      todos = JSON.parse(localStorage.getItem('todos'));
+    }
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
+  function getTodos() {
+    if (localStorage.getItem('todos') === null) {
+      todos = [];
+    } else {
+      todos = JSON.parse(localStorage.getItem('todos'));
+    }
+
+    todos.map((todo) => {
+      const newTodo = document.createElement('li');
+      newTodo.classList.add('task');
+      newTodo.setAttribute('id', todo.index);
+      if (todo.completed === true) newTodo.classList.add('completed');
+      const completedCheck = document.createElement('INPUT');
+      completedCheck.setAttribute('type', 'checkbox');
+      completedCheck.setAttribute('id', todo.index);
+      if (todo.completed === true) completedCheck.setAttribute('checked', true);
+      completedCheck.classList.add('check');
+      newTodo.appendChild(completedCheck);
+      const textDescription = document.createElement('INPUT');
+      textDescription.setAttribute('type', 'text');
+      textDescription.setAttribute('value', todo.description);
+      textDescription.classList.add('task-des');
+      textDescription.setAttribute('id', todo.index);
+      newTodo.appendChild(textDescription);
```

Comment:
Hi, in this line, you could use a <p> tag instead of an input, if you add to the paragraph the attribute "contenteditable", you can change it later, and it's easier for the browser to render the paragraphs than the inputs!

Reply:
ok thank you for the review will work on it